classes/01class/exercises/c01-git03/starkmatt/c01-e03.txt

```diff
@@ -0,0 +1,47 @@
+Created a seperate local branch called git-e03 and created the readme file as well as the large file
```

Comment:
I think re have a duplicated exercise answer here.

There is `classes/01class/exercises/c01-git03/starkmatt/c01-e03.txt` and `classes/01class/exercises/c01-git03/starkmatt/c01-git03.txt`.

We need to review how you are pushing branches and adding exercises.

Happy to check it with you after class next week.

Reply:
Thanks Denis,
Agreed, I'm always finding myself going back and fixing up commits / branches because it's pushing files that I haven't intentionally created.
My gut feeling tells me it's where I'm creating my branches initially meaning that the location I'm in (academy/. instead of navigating to the location where I'm adding the changes such as  /classes/01class/exercises/c01-git03/starkmatt and from there creating the branch)

Reply:
Further to this,
I realized what I've done wrong.
Made changes to the master, and put this second branch I created !